docs/dev-guide/installingNative.md

```diff
@@ -22,7 +22,11 @@ Most of these prerequisites can be installed using the package manager of your O
     On Ubuntu, use `apt` for the main prerequisites:
 
     ```sh
-    sudo apt install git gcc libc6-dev graphviz libgraphviz-dev redis postgresql postgresql-contrib postgresql-server-dev-all
+    sudo apt install -y git gcc libc6-dev graphviz libgraphviz-dev redis postgresql-common
+    # Configure Postgres repository
+    sudo /usr/share/postgresql-common/pgdg/apt.postgresql.org.sh
+    sudo apt-get update
```

Comment:
Nit: just checked the script itself, it already does the `apt-get update`, so this line is not strictly needed.

Reply:
Updated in a4071f6fd2aab15d9dfeb0a451cba59603861d17. Also switched back to the interactive version of apt.